src/utils/BlockExplorerUtils.ts

```diff
@@ -27,17 +31,21 @@ export function resolveBlockExplorerDomain(networkId: number): string {
  * @returns A formatted markdown block explorer link to the given transaction hash or address on the given chainId.
  */
 function _createBlockExplorerLinkMarkdown(hex: string, chainId = 1): string | null {
-  if (hex.substring(0, 2) != "0x") {
-    return null;
-  }
-  const shortURLString = createShortHexString(hex);
-  // Transaction hash
-  if (hex.length == 66) {
-    return `<${resolveBlockExplorerDomain(chainId)}/tx/${hex}|${shortURLString}>`;
-  }
-  // Account
-  else if (hex.length == 42) {
-    return `<${resolveBlockExplorerDomain(chainId)}/address/${hex}|${shortURLString}>`;
+  try {
+    if (hex.substring(0, 2) != "0x") {
+      return null;
+    }
+    const shortURLString = createShortHexString(hex);
+    // Transaction hash
+    if (hex.length == 66) {
+      return `<${resolveBlockExplorerDomain(chainId)}/tx/${hex}|${shortURLString}>`;
+    }
+    // Account
+    else if (hex.length == 42) {
+      return `<${resolveBlockExplorerDomain(chainId)}/address/${hex}|${shortURLString}>`;
+    }
+  } catch (e) {
+    throw new Error(`Could not create block explorer link. ChainId: ${chainId}, Hex: ${hex}`);
   }
   return null;
```

Comment:
Small proposal to avoid it blowing up when we're adding new chains:

```suggestion
  const unsupported = `<unsupported hash (${hex})>`;
  if (hex.substring(0, 2) != "0x") {
    return unsupported;
  }
  
  try {
    const shortURLString = createShortHexString(hex);
    // Transaction hash
    if (hex.length == 66) {
      return `<${resolveBlockExplorerDomain(chainId)}/tx/${hex}|${shortURLString}>`;
    }
    // Account
    else if (hex.length == 42) {
      return `<${resolveBlockExplorerDomain(chainId)}/address/${hex}|${shortURLString}>`;
    }
  } catch (e) {
    return `<unsupported chainId (${chainId}), hash: ${hex}>`;
  }
  return unsupported;
```